services/node_stake_update.proto

```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
    */
   Timestamp end_of_staking_period = 1;
 
+  /**
+   * The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day.
+   */
+  int64 max_daily_reward_rate = 2;
+
+  /**
+   * The fraction of the network and service fees (as a whole number in the range [0, 100]) paid to 
+   * the node reward account 0.0.801.
+   */
+  int32 node_reward_fees_percent = 3;
+
   /**
    * Staking info of each node at the beginning of the new staking period
    */
-  repeated NodeStake node_stake = 2;
+  repeated NodeStake node_stake = 4;
+
+  /**
+   * The maximum number of trailing periods for which a user can collect rewards. For example, if this
+   * is 365 with a UTC calendar day period, then users must collect rewards at least once per calendar
+   * year to avoid missing any value.
+   */
+  int64 num_rewardable_trailing_periods = 5;
+
+  /**
+   * The number of minutes in a staking period. Note for the special case of 1440 minutes, periods are 
+   * treated as UTC calendar days, rather than repeating 1440 minute periods left-aligned at the epoch.
+   */
+  int64 period_mins = 6;
+
+  /**
+   * The fraction of the network and service fees (as a whole number in the range [0, 100]) paid to 
+   * the staking reward account 0.0.800.
+   */
+  int32 staking_reward_fees_percent = 7;
+
+  /**
+   * The minimum balance of staking reward account 0.0.800 required to active rewards.
+   */
+  int64 start_threshold = 8;
+
+  /**
+   * The total number of tinybars to be distributed as staking rewards each period.
+   */
+  int64 total_rewards_per_period = 9;
```

Comment:
Q: is this just the sum of `node_stake.reward_rate`? Or `rewardRate`, a.k.a, 

`StakingRewardRate - the total tinybars earned as staking reward during each staking period (example: 100_000_000L * 1_000_000_000 / 365 to emit rewards at a total rate of one billion hbars per year)
` in the HIP?

Reply:
It's the `StakingRewardRate` from the HIP. 👍